bin/run_ccm_performance_test.sh

```diff
@@ -147,6 +147,13 @@ ccm node1 cqlsh -u cassandra -p cassandra -x "ALTER ROLE cassandra WITH OPTIONS
 run_stress authentication-authorization-audit-role-whitelist-permission-derived
 stop_cassandra
 
+create_cluster vanilla-chron
+${SCRIPT_PATH}/configure_ccm_querylog_chronicle.sh
+start_cassandra
+create_dummy_whitelists
+run_stress vanilla-querylog-chronicle
```

Comment:
The numbers for this doesn't appear to be in the stress graphs?

Reply:
Once this PR is merged the links will be correct. Until then:
https://rawgit.com/eperott/ecaudit/query_logger/doc/ecaudit-performance.html

Reply:
My bad, I missed that the link was pointing to master!